src/polish/ParadigmsPol.gf

```diff
@@ -1,28 +1,386 @@
+        <_ + "a",_ + "ów"> => NM.mkNTable0501 sgnom ;
+        <_ + "a",_ + "ej"> => NM.mkNTable0013 sgnom ;  -- Alternatives: mkNTable0504,mkNTable0755
+        <_ + "a",_ + "iej"> => NM.mkNTable0283 sgnom ;
+        <_ + "a",_ + "ego"> => NM.mkNTable0614 sgnom ;  
+        <_ + "ć",_ + "cia"> => NM.mkNTable0069 sgnom ; -- Alternatives: mkNTable0573,mkNTable0923,mkNTable0922,mkNTable0838,mkNTable0649,mkNTable0734,mkNTable0794,mkNTable0793
+        <_ + "ć",_ + "ci"> => NM.mkNTable0475 sgnom ; -- Alternatives: mkNTable0567,mkNTable1014,mkNTable0792,mkNTable0814
+        <_ + "ć",_ + ""> => NM.mkNTable0069 sgnom ; -- Alternatives: 
+        <_ + "e",_ + ""> => NM.mkNTable0107 sgnom ; 
+        <_ + "e",_ + "a"> => NM.mkNTable0413 sgnom ; -- Alternatives: mkNTable0477,mkNTable0836,mkNTable0553
+        <_ + "e",_ + "e"> => NM.mkNTable0438 sgnom ; -- Alternatives: mkNTable0527,mkNTable0963,mkNTable0646,mkNTable0714
+        <_ + "e",_ + "i"> => NM.mkNTable0081 sgnom ; -- Alternatives: 
+        <_ + "e",_ + "u"> => NM.mkNTable0715 sgnom ; -- Alternatives: 
+        <_ + "e",_ + "y"> => NM.mkNTable0311 sgnom ; -- Alternatives: 
+        <_ + "e",_ + "ów"> => NM.mkNTable0214 sgnom ; -- Alternatives: mkNTable0764
+        <_ + "e",_ + "ego"> => NM.mkNTable0472 sgnom ; -- Alternatives: mkNTable0554,mkNTable0694
+        <_ + "e",_ + "ych"> => NM.mkNTable0508 sgnom ;
+        <_ + "o",_ + "a"> => NM.mkNTable0079 sgnom ; -- Alternatives: mkNTable0158, mkNTable0205,mkNTable0250,mkNTable0265,mkNTable0295,mkNTable0332,mkNTable0388,mkNTable1012,mkNTable1013,mkNTable0618
+        <_ + "o",_+"o"> => NM.mkNTable0162 sgnom ; -- Alternatives: mkNTable0182,mkNTable0217,mkNTable0403,mkNTable0918,mkNTable0962
+        <_ + "o",_ + "y"> => NM.mkNTable0239 sgnom ;
+         -- Alternatives: mkNTable0239
+        <_ + "o",_ + "ina"> => NM.mkNTable0083 sgnom ; 
+        <_,_ + "a"> => NM.mkNTable0503 sgnom ; -- Alternatives: mkNTable0944,mkNTable0497,mkNTable0244,mkNTable0350,mkNTable0282,mkNTable0197,mkNTable0131
+        <_,_ + "i"> => NM.mkNTable0995 sgnom ; -- Alternatives: mkNTable0995,mkNTable0583
+        <_,_ + "u"> => NM.mkNTable0171 sgnom ; -- Alternatives: mkNTable0696,mkNTable0111,mkNTable0539,mkNTable0247
+        <_,_ + "y"> => NM.mkNTable0550 sgnom ;
+        <_,_ + "ia"> => NM.mkNTable0803 sgnom  ;-- Alternative: mkNTable0648
+        <_,_ + "iu"> => NM.mkNTable0662 sgnom ;
+        <_,_ + "na"> => NM.mkNTable0286 sgnom ;
+        <_,_ + "ego"> => NM.mkNTable0589 sgnom ; -- Alternative: mkNTable0966
+        <_,_> => guess_paradigm_basic sgnom}; 
+     -- 1 string
+     guess_paradigm : (sgnom : Str) -> SubstForm => Str = guess_paradigm_basic 
```

Comment:
There's a mistake here! Will push a fix.

Reply:
Sorry, it's not a mistake, just [eta-reduction](https://wiki.haskell.org/Eta_conversion). But I understand if it confuses people, so feel free to fix it to 

```haskell
guess_paradigm : (sgnom : Str) -> SubstForm => Str = \sgnom -> guess_paradigm_basic sgnom
```